# Generated by Django 5.1.3 on 2024-11-22 10:43

import varda.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('varda', '0092_errorreports'),
    ]

    operations = [
        migrations.AlterField(
            model_name='historicaltoimipaikka',
            name='kayntiosoite_postinumero',
            field=models.CharField(max_length=10, validators=[varda.validators.validate_postinumero_toimipaikka]),
        ),
        migrations.AlterField(
            model_name='historicaltoimipaikka',
            name='postinumero',
            field=models.CharField(max_length=20, validators=[varda.validators.validate_postinumero_toimipaikka]),
        ),
        migrations.AlterField(
            model_name='toimipaikka',
            name='kayntiosoite_postinumero',
            field=models.CharField(max_length=10, validators=[varda.validators.validate_postinumero_toimipaikka]),
        ),
        migrations.AlterField(
            model_name='toimipaikka',
            name='postinumero',
            field=models.CharField(max_length=20, validators=[varda.validators.validate_postinumero_toimipaikka]),
        ),
    ]
